compliance_checker/acdd.py

```diff
@@ -69,7 +69,7 @@ def check_recommended(self, ds):
     def check_suggested(self, ds):
         return [
             'contributor_name',
-            ('contributor_role', ['principalInvestigator', 'author']),
+            'contributor_role',
```

Comment:
@daf , @lukecampbell , Which ACDD version is checker currently checking against? 1.1, 1.2, or 1.3?